subxt/src/backend/unstable/rpc_methods.rs

```diff
@@ -0,0 +1,890 @@
+// Copyright 2019-2023 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! An interface to call the  API methods. See
+//! <https://github.com/paritytech/json-rpc-interface-spec/> for details of the API
+//! methods exposed here.
+
+use crate::backend::rpc::{rpc_params, RpcClient, RpcSubscription};
+use crate::{Config, Error};
+use futures::{Stream, StreamExt};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::task::Poll;
+
+/// An interface to call the unstable RPC methods. This interface is instantiated with
+/// some `T: Config` trait which determines some of the types that the RPC methods will
+/// take or hand back.
+pub struct UnstableRpcMethods<T> {
+    client: RpcClient,
+    _marker: std::marker::PhantomData<T>,
+}
+
+impl<T> Clone for UnstableRpcMethods<T> {
+    fn clone(&self) -> Self {
+        Self {
+            client: self.client.clone(),
+            _marker: self._marker,
+        }
+    }
+}
+
+impl<T> std::fmt::Debug for UnstableRpcMethods<T> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("UnstableRpcMethods")
+            .field("client", &self.client)
+            .field("_marker", &self._marker)
+            .finish()
+    }
+}
+
+impl<T: Config> UnstableRpcMethods<T> {
+    /// Instantiate the legacy RPC method interface.
+    pub fn new(client: RpcClient) -> Self {
+        UnstableRpcMethods {
+            client,
+            _marker: std::marker::PhantomData,
+        }
+    }
+
+    /// Subscribe to `chainHead_unstable_follow` to obtain all reported blocks by the chain.
+    ///
+    /// The subscription ID can be used to make queries for the
+    /// block's body ([`chainhead_unstable_body`](UnstableRpcMethods::chainhead_unstable_follow)),
+    /// block's header ([`chainhead_unstable_header`](UnstableRpcMethods::chainhead_unstable_header)),
+    /// block's storage ([`chainhead_unstable_storage`](UnstableRpcMethods::chainhead_unstable_storage)) and submitting
+    /// runtime API calls at this block ([`chainhead_unstable_call`](UnstableRpcMethods::chainhead_unstable_call)).
+    ///
+    /// # Note
+    ///
+    /// When the user is no longer interested in a block, the user is responsible
+    /// for calling the [`chainhead_unstable_unpin`](UnstableRpcMethods::chainhead_unstable_unpin) method.
+    /// Failure to do so will result in the subscription being stopped by generating the `Stop` event.
+    pub async fn chainhead_unstable_follow(
+        &self,
+        runtime_updates: bool,
```

Comment:
nit: I think this is now named `with_runtime` from [spec](https://github.com/paritytech/json-rpc-interface-spec/blob/main/src/api/chainHead_unstable_follow.md)

Reply:
Yup; I renamed it :)